src/cr/cube/matrix.py

```diff
@@ -26,11 +26,33 @@ def __init__(self, unordered_matrix):
 
     @classmethod
     def matrix(cls, cube, dimensions, slice_idx):
-        """Return a TransformedMatrix object constructed from this cube result."""
+        """ -> TransformedMatrix object constructed from values of `cube`.
+
+        `cube` is the `cr.cube.Cube` object containing the data for this matrix. Note
+        that not all the data in `cube` will necessarily be used by this matrix. When
+        `cube` is more than 2-dimensional, it is "sliced" and each slice gets its own
+        matrix (and `_Slice` object).
+
+        `dimensions` is a pair (2-tuple) of (rows_dimension, columns_dimension)
+        Dimension objects. These are based on two of the dimensions of `cube` but may
+        not be the first two and may and often do have transformations applied that are
+        not present on the `cube` dimensions from which they derive.
+
+        `slice_idx` is an int offset indicating which portion of `cube` data to use for
+        this matrix.
```

Comment:
A trick from an ex UX colleague - don't leave the orphan words for new line - it's mostly for when you leave a single word for the new line, that it gets skipped in reading. Here we have two words, but still, if you divided the sentence into `". . . of cube data"` and `"to use for this matrix."`, it'd be more balanced across lines (even though further away from the 88 limit).